x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/dimension_editor.tsx

```diff
@@ -215,13 +221,38 @@ export function PopoverEditor(props: PopoverEditorProps) {
           );
         },
       };
-    });
-  }
+    }
+  );
 
   return (
-    <div id={columnId} className="lnsIndexPatternDimensionEditor">
-      <EuiFlexGroup gutterSize="s" direction="column">
-        <EuiFlexItem>
+    <div id={columnId}>
+      <div className="lnsIndexPatternDimensionEditor__section lnsIndexPatternDimensionEditor__section--shaded">
+        <EuiFormLabel>Choose a function</EuiFormLabel>
+        <EuiSpacer size="s" />
+        <EuiListGroup
+          className={sideNavItems.length > 3 ? 'lnsIndexPatternDimensionEditor__columns' : ''}
+          gutterSize="none"
+          listItems={sideNavItems}
+          maxWidth={false}
+        />
```

Comment:
The hover state and selected state for this list are using the same background color, such as in this example:
 
<img width="420" alt="Screen Shot 2020-09-09 at 5 58 40 PM" src="https://user-images.githubusercontent.com/666475/92660269-1650da80-f2c8-11ea-91fe-7e09a641c767.png">

Reply:
Yeah, this is just how EuiListGroup handles these states, but the dark bold version is really what's signifying selection fo rus.